Assets/Scripts/UI/InGame/Graphics/QualitySwitcher.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Assets.Scripts.UI.InGame
+{
+	public class QualitySwitcher : MonoBehaviour {
+		public Text Label;
+		public Slider Slider;
+
+		private void Update() {
+			Label.text = QualitySettings.names[QualitySettings.GetQualityLevel()];
+			// this was causing the bug // gameObject.GetComponentInChildren<Slider>().value = QualitySettings.GetQualityLevel();
```

Comment:
Okay, so we can remove this comment. Also, do we need to do this check every frame?